build/commands/lib/config.js

```diff
@@ -848,6 +848,12 @@ Config.prototype.update = function (options) {
   if (options.target) {
     this.buildTarget = options.target
   }
+  if (options.use_pgo) {
```

Comment:
`|| is_official_build`

Reply:
The idea is that we want to enable PGO only for certain builds. Right now it's only public nightly CI builds, not beta and stable.
We also not want to enable it for local build.